perfkitbenchmarker/linux_benchmarks/fio_benchmark.py

```diff
@@ -394,7 +396,9 @@ def GetLogFlags():
                    (FLAGS.fio_iops_log, '--write_iops_log=%(filename)s',),
                    (collect_logs, '--log_avg_msec=%(interval)d',)]
   fio_command_flags = ' '.join([flag for given, flag in fio_log_flags if given])
-  return fio_command_flags % {'filename': PKB_FIO_LOG_FILE_NAME,
+  now = time.time()
+  filename_base = '%s_%s' % (PKB_FIO_LOG_FILE_NAME, str(now))
```

Comment:
I don't feel strongly about it, I guess one alternative would be to convert it to nanoseconds and use that as an int, that way you don't lose the precision and you can use an integer.

If not, this is good enough.